db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -2589,12 +2589,14 @@ TOKEN [IGNORE_CASE] :
 |	<AND: "and">
 |	<ANY: "any">
 |	<ARE: "are">
+|	<ARRAY: "array">
 |	<AS: "as">
 |	<ASC: "asc">
 |	<ASSERTION: "assertion">
 |	<AT: "at">
 |	<AUTHORIZATION: "authorization">
 |	<AVG: "avg">
+|	<B: "b">
```

Comment:
We should put "b", "L", "T" in the non-reserved keyword section.

Reply:
Moved them to non-reserved keywords section.